definitions/pipelines/germline_exome_hla_typing.cwl

```diff
@@ -0,0 +1,174 @@
+    picard_metric_accumulation_level:
+        type: string
+    emit_reference_confidence:
+        type: string
+    gvcf_gq_bands:
+        type: string[]
+    intervals:
+        type:
+            type: array
+            items:
+                type: array
+                items: string
+    vep_cache_dir:
+        type: string
+    synonyms_file:
+        type: File?
+    annotate_coding_only:
+        type: boolean?
+    custom_gnomad_vcf:
+        type: File?
+        secondaryFiles: [.tbi]
+    qc_minimum_mapping_quality:
+        type: int?
+    qc_minimum_base_quality:
+        type: int?
+    custom_clinvar_vcf:
+        type: File?
+        secondaryFiles: [.tbi]
+    optitype_name:
+        type: string?
+outputs:
+    cram:
+        type: File
+        outputSource: germline_exome_wf/cram
+    mark_duplicates_metrics:
+        type: File
+        outputSource: germline_exome_wf/mark_duplicates_metrics
+    insert_size_metrics:
+        type: File
+        outputSource: germline_exome_wf/insert_size_metrics
+    insert_size_histogram:
+        type: File
+        outputSource: germline_exome_wf/insert_size_histogram
+    alignment_summary_metrics:
+        type: File
+        outputSource: germline_exome_wf/alignment_summary_metrics
+    hs_metrics:
+        type: File
+        outputSource: germline_exome_wf/hs_metrics
+    per_target_coverage_metrics:
+        type: File[]
+        outputSource: germline_exome_wf/per_target_coverage_metrics
+    per_target_hs_metrics:
+        type: File[]
+        outputSource: germline_exome_wf/per_target_hs_metrics
+    per_base_coverage_metrics:
+        type: File[]
+        outputSource: germline_exome_wf/per_base_coverage_metrics
+    per_base_hs_metrics:
+        type: File[]
+        outputSource: germline_exome_wf/per_base_hs_metrics
+    summary_hs_metrics:
+        type: File[]
+        outputSource: germline_exome_wf/summary_hs_metrics
+    flagstats:
+        type: File
+        outputSource: germline_exome_wf/flagstats
+    verify_bam_id_metrics:
+        type: File
+        outputSource: germline_exome_wf/verify_bam_id_metrics
+    verify_bam_id_depth:
+        type: File
+        outputSource: germline_exome_wf/verify_bam_id_depth
+    gvcf:
+        type: File[]
+        outputSource: germline_exome_wf/gvcf
+    final_vcf:
+        type: File
+        outputSource: germline_exome_wf/final_vcf
+        secondaryFiles: [.tbi]
+    coding_vcf:
+        type: File
+        outputSource: germline_exome_wf/coding_vcf
+        secondaryFiles: [.tbi]
+    limited_vcf:
+        type: File
+        outputSource: germline_exome_wf/limited_vcf
+        secondaryFiles: [.tbi]
+    vep_summary:
+        type: File
+        outputSource: germline_exome_wf/vep_summary
+    optitype_tsv:
+        type: File
+        outputSource: optitype/optitype_tsv
+    optitype_plot:
+        type: File
+        outputSource: optitype/optitype_plot
+steps:
+    germline_exome_wf:
```

Comment:
Echoing [a comment](https://github.com/genome/analysis-workflows/pull/627#discussion_r276324863) comment I left on #627; I generally think we should leave off tags like `wf` from the names of steps.

Reply:
Fixed in 3709c4a